internal/engine/engine.go

```diff
@@ -220,7 +220,7 @@ func (c *craneEngine) ExecuteChecks(ctx context.Context) error {
 		c.results.TestedImage = c.image
```

Comment:
Assuming you make the change to the import because of the name of this very temporary variable. I'd rather see this change rather than the import.

Reply:
Agreed - change this and leave the import. It's a smaller change.

Reply:
It's actually not for this, it's for the code on line 223, since there `check` is a scoped struct as part of a for loop.

Reply:
I was referring to the name of the var in the for loop.